trillian/ctfe/handlers_test.go

```diff
@@ -617,7 +617,7 @@ func TestGetSTH(t *testing.T) {
 			descr:  "backend-failure",
 			rpcErr: errors.New("backendfailure"),
 			want:   http.StatusInternalServerError,
```

Comment:
This is coincidentally the same status. Is it easy to add a test that shows the behaviour change?

Reply:
Good idea, done.